### PR TITLE
Update leader election attributes for kube-scheduler and kube-control…

### PR DESCRIPTION
**Pull request by shalver**
…ler-manager

Issue: https://github.ibm.com/alchemy-containers/armada-update/issues/2328

### CODE REVIEW

File: assets/kube-controller-manager/config.yaml [hunk 1]
```diff
@@ -40,8 +40,12 @@ extendedArguments:
   - configmaps
   leader-elect:
   - 'true'
+  leader-elect-lease-duration:
+  - 60s
+  leader-elect-renew-deadline:
+  - 15s
   leader-elect-retry-period:
-  - 3s
+  - 10s
```
Comment (rtheis) [inline assets/kube-controller-manager/config.yaml:48]: It think this is too high and yields zero seconds of Kubernetes API server downtime per https://github.com/openshift/library-go/blob/4b9033d00d37b88393f837a88ff541a56fd13621/pkg/config/leaderelection/leaderelection.go#L90-L95.  It think we should probably use `3s` here.  Then maybe we can bump further in a OpenShift 4.9.

**Review by rtheis — CHANGES_REQUESTED** [review]

File: assets/kube-controller-manager/config.yaml (hunk 1 above)
Comment (shalver) [reply assets/kube-controller-manager/config.yaml]: retry-period updated to 3s

**Review by rtheis — APPROVED** [review]
/lgtm
/approved

**Pull request merged**